### ISSUE TITLE
Update Project Plan

### ISSUE THREAD

**Issue by enisimsar** [bounswe/bounswe2018group5#35]
The project plan will be updated according to customer review.

- [x] No need for writing down the meetings within the project plan. It is within the scope of the communication plan.
- [x] It is nice to have milestones, but you need to place them in the appropriate places. Or in other words, sort the project plan according to “start time”. For instance, Task1, Task2, Task3… Task M, then Milestone 1. This means that Task1 to M are the deliverables of the Milestone 1. After Milestone1, Task M+1,…. Task N, and Milestone 2, which means that Task M+1 to N are the deliverables of the Milestone 2. With the current format, it is challenging to see the deliverables of the Milestone 2. 
- [x] It would be nice to add some “revision” tasks into your plan, such as “revising the requirements” before the semester ends.
- [x] Please determine the dependencies among the tasks. Which tasks should be completed before starting the task X?
- [x] What about the member assignments for the future tasks? You need to assign some members for the tasks until this semester. This is the main objective of the project plan, not just timing.
